lib/board.rb

```diff
@@ -1,5 +1,8 @@
 
 class Board
+  
+    attr_accessor :moves
+  
     def display
         "    1    |  2   |   3   #{horizontal_bar}    4    |  5   |   6   #{horizontal_bar}    7    |  8   |   9  \n"
     end
```

Comment:
We discussed before that Board should not be responsible for displaying itself. That responsibility should live in another class.

Reply:
I'll make a new class for rendering the board.